### DISCUSSION TITLE
Defining "beginner-friendly"

### DISCUSSION THREAD

**Discussion by lpatmo** [codebuddies/frontend#136] [Ideas]
A few initial thoughts for what we could tell the user when they try to check something as "beginner-friendly":

What does "beginner-friendly" mean?
- tutorial is easy-to-replicate
- tagged as "intro"

**Comment by BethanyG** [comment]
-  technical terms are clearly defined (_preferably in "non technical person terms"_), or links to definitions, documentation, and resources are provided.
-  any concepts or techniques beyond "core" programming knowledge are explained/demonstrated
-  context/background for the method/library/process is provided
- "next steps" are given for deepening knowledge
